### ISSUE TITLE
Align and clarify on focus for Equity Framework for Project Selection

### ISSUE THREAD

**Issue by celestemespinoza** [usds/equity_practice#19]
## Description
Equity Delivery met with leadership at the onsite and identified a focus on project selection where they would appreciate our attention. The team also met with our sibling A11Y team who are investigating how to incorporate accessibility considerations within project selection. Together, both teams need to align on the common objective for this work. 

Equity Delivery also needs to align on pre-work for continuing the conversation with Mina with respect to the criteria and nuance used to determine project selection. She would like to go through the submissions together and discuss approvals, follow-ups, and rejections.

### Acceptance Criteria
- Understand A11Y teams objective for this work
- Schedule time to align with A11Y team around project selection
- Provide a short form definition for equity based on current documentation for alignment with Leadership
- Clarify pre-work items we would like to ask for or provide during our follow-up time with Leadership
- Reach out to Dayna once all of this has been defined to set up time with Mina

## Tasks
a really loose place for us to draft up tasks and assign them accordingly

- [ ] a task for @celeste
- [ ] a task for @celeste
- [ ] a task for @celeste

## Project Artifacts

**Comment by jeremyzitomer-usds** [comment]
Copying over from slack conversation, @celestemespinoza said on 12/15:
> Our asks for the outstanding item around project selection is in the ticket but should be at least 6-10 projects that meet the following criteria:
> Projects declined but with an ask for additional research and/or review
> Projects declined because it didn't meet minimum threshold for criteria set out in the rubric (no requests for research or review)
> Projects accepted but with an ask for additional research and/or review
> Projects that were accepted that did not came from a source that is not the Executive Team
> Projects that were accepted that came from the Executive Team
> Any other projects that they think might bare light on how project selection occurs so we understand it better
> We should also be asking for dates in late January to allocate enough time for everyone to attend a review (Mina, Cori, Ankit,...) of at least 60 minutes, preferably 90.
> We should anticipate talking about this for 10 minutes max to make sure we're aligned on this still and that Mina, et al don't have asks from us ahead of this workshop.

**Comment by jeremyzitomer-usds** [comment]
@celestemespinoza, I'd like to get insight both on a) the thought process underlying the project selection decisions they make (which your super helpful categories will shed light on), in addition to b) the relative volume of each of these categories. For that reason, my proposal (pending agreement from Mina and Joe) is that we:
- ask for a copy of all project submissions they received over a 30 day window
- which should include at least one of all of the above categories
- ideally, if they have a central project selection documentation place where they already record information like the above categories, i think we should ask to get that from them too (assuming Joe et al can share it without too big a lift).

Given the importance of this workstream, I think this is a reasonable ask. If we're comfortable with this, would recommend we convey this to Mina in our conversation today. Thoughts?

cc @natasha-jamal @AlexBornkesselUSDS

**Comment by AlexBornkesselUSDS** [comment]
Chiming - I prefer @celestemespinoza's original approach (which I copied into a slide, in case something concrete can help leadership get there quicker). 

To me, the risk w/ the revised approach @jeremyzitomer-usds proposed could lead to no sharing of anything (vs. sharing of something). It could also be taken as more confrontational? or risk-based. If we get the something and we still have questions, we could then make another ask. 

[slide.pptx](https://github.com/usds/equity_practice/files/13717353/slide.pptx)

**Comment by jeremyzitomer-usds** [comment]
I'm okay to disagree and go with the approach @celestemespinoza and @AlexBornkesselUSDS y'all are suggesting. I do worry we won't feel like we have enough information to learn whether/how we might adjust the process, but I'm okay to iterate on it, assuming the leadership team is willing to have subsequent meetings with us about this.

Thanks for the slide @AlexBornkesselUSDS , fixed some small typos for when we present to leadership:
[slide_jz-edits.pptx](https://github.com/usds/equity_practice/files/13717434/slide_jz-edits.pptx)

**Comment by celestemespinoza** [comment]
@jeremyzitomer-usds I agree with @AlexBornkesselUSDS that some of this feels a little risky for this audience but I want to clarify and meet somewhere in the middle.

Questions coming back:
1. why is the volume important? if we only received 6 in the last 30 days, are we good with just the six or are you trying to get at a larger volume to just have a larger representative sample? If it's the latter, then I think we should address the total # and the timebox.

 - 60 days
 - 2-5 per category
 - a total # of submissions per category so we can assess volume

 Yes, this means we'd be asking for more, but if the issue is representative sample, then let's try to get to that. We could also work backwards and ask for total number of submissions over the year, median # of submissions per month to account for high volume months, and then adjust the number from there.

 ** We can work with Joe to identify those he thought were good candidates too. He seemed open to that.

 ** We should prepare to answer on why the volume needed for early assessment. Our curiosity isn't an answer.

2. I think we can ask the review the project selection spreadsheet but asking for it outright without guidance or context feels like a lot. I think we can request time apart with Joe to screenshare with us, and ask for view only access. We already know that they capture this in a spreadsheet, but we might not understand how data gets captured, what gets captured, or even what is not captured. We aren't likely to understand that in its entirety from just looking at the spreadsheet, or from looking at it for 5 minutes, but since Joe will be there today it sounds like a reasonable ask to get 20-30 minutes with him to help orient us on the spreadsheet so we can also work with it in workshop appropriately.

They are also going to want a pre-read on what we plan to ask so we should start thinking about what that could look like.

**Comment by celestemespinoza** [comment]
@AlexBornkesselUSDS The only thing I would change about the slide is providing some recommended dates. 

Maybe we can recommend the week of January 22, January 25 or 26th? That gives enough time to pull everything together. Yes?

**Comment by AlexBornkesselUSDS** [comment]
Updated slide attached
[slide_jz-edits_AB.pptx](https://github.com/usds/equity_practice/files/13718079/slide_jz-edits_AB.pptx)

**Comment by celestemespinoza** [comment]
via Joe:
this is very round numbers...
I think on avg, we might get about 6-8 requests per month.  so call it a range of 70-100 requests over the year
I've been here half the year, and really only seen us greenlight ~7 of those net new things.  There were 2 others we said yes to, then they fizzled, and we cancelled early.
If that 7 per 6 months trend were the same over the first half of the year, maybe we greenlight 10-20% of all requests over a year.
I don't think we auto-reject anyone.  We do tend to take all the intake requests as meetings, even if it lands in a no, or we'll give some other form of very narrow follow-up (e.g. a convo with one of our staff)


as I told you in the coffee shop, I think this is probably not the most productive approach, honestly.  even if you get some head nodding today, I think a better time spend for the moment is how to integrate into projects in their discovery/research phase, for both new projects and projects pivoting-- as there is actually a lot of that, and it's easier to influence

**Comment by celestemespinoza** [comment]
**Notes**
- Projects are being collected by David Pham for us
- We wil go into this with loose discovery in mind
- We will address formal equity in project delivery after we meet with them
- Schedule for 60 minutes, third/fourth week of January 24

**Comment by AlexBornkesselUSDS** [comment]
Pinged Joe and David via Slack, to check-in on status and ETA for when info can be shared
